lib/icingadb/icingadb-objects.cpp

```diff
@@ -145,12 +145,8 @@ void IcingaDB::UpdateAllConfigObjects()
 		UserGroup::TypeInstance,
 		Zone::TypeInstance
 	}) {
-		ConfigType *ctype = dynamic_cast<ConfigType *>(type.get());
-		if (!ctype)
-			continue;
```

Comment:
When removing this part here, why not just make `types` a `std::vector<Type::Ptr>` and move the conversion to lower case to lambda below? Allows to avoid this for loop and fiddling around with `.first` and `.second` below.